index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Conference page: Creative Art Summit</title>
+<link rel="stylesheet" href="style.css">
+</head>
+<body>      
+    <header>
+        <div class="social">
+     <ul class="gray">
+        <li>
+            <a href="#"><img src="Images/2Vector.png" alt="#" class="tweet"></a>
+          </li>
+         <li>English</li>
+         <li>My page</li>
+         <li>Logout</li>
+     </ul>
```

Comment:
- [ ][OPTIONAL] I suggest you to add some `padding` on the right side of `logout`:

Current design  | Design with padding
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/147923241-e8cee1a3-ff8e-4f23-b26f-7664132b058e.png) | ![image](https://user-images.githubusercontent.com/77038610/147923211-8faf09d5-1f7c-4936-8ae5-9f35cbffca04.png)